### ISSUE TITLE
JS error on submit Login

### ISSUE THREAD

**Issue by VardgesK** [orchidsoftware/platform#2892]
Hi @tabuna .

After submitting the login button, I'm encountering a JavaScript error.

```
orchid.js?id=f5d75d25a52cfc53164568a973a41033:16 Uncaught TypeError: Cannot read properties of null (reading 'value')
    at orchid.js?id=f5d75d25a52cfc53164568a973a41033:16:86940
    at b (orchid.js?id=f5d75d25a52cfc53164568a973a41033:2:50059)
    at #e (orchid.js?id=f5d75d25a52cfc53164568a973a41033:2:55218)
    at $.perform (orchid.js?id=f5d75d25a52cfc53164568a973a41033:2:54278)
    at et.start (orchid.js?id=f5d75d25a52cfc53164568a973a41033:2:58625)
    at Kt.submitForm (orchid.js?id=f5d75d25a52cfc53164568a973a41033:16:20505)
    at Object.formSubmitted (orchid.js?id=f5d75d25a52cfc53164568a973a41033:16:46373)
    at HTMLDocument.submitBubbled (orchid.js?id=f5d75d25a52cfc53164568a973a41033:2:62961)
```

Screenshots

![Screenshot 2024-09-03 093102](https://github.com/user-attachments/assets/7b5f6e26-ec26-4b33-9ded-f34160e248a4)
![Screenshot 2024-09-03 093117](https://github.com/user-attachments/assets/14e31fc1-799e-4afe-b406-7e7be568ce5f)

**Comment by tabuna** [comment]
Should be fixed with the release of 14.33.1.

**Comment by VardgesK** [comment]
Yes it works, thanks
